html/modules/custom/docstore/src/Controller/VocabularyController.php

```diff
@@ -632,7 +632,13 @@ protected function createTermFromUserParameters($params) {
   /**
    * Get term.
    */
-  public function getTerm($id, $term_id) {
+  public function getTerm($id, $term_id, Request $request) {
+    // Parse JSON.
+    $params = json_decode($request->getContent(), TRUE);
+    if (empty($params) || !is_array($params)) {
```

Comment:
This should allow no params at all, https://travis-ci.com/github/UN-OCHA/docstore-site/builds/215220585#L3851

We don't need params for getting a term

Reply:
FYI, I merged https://github.com/UN-OCHA/docstore-site/pull/75